identity-service/src/routes/authentication.js

```diff
@@ -25,8 +25,31 @@ module.exports = function (app) {
    * authentication process
    */
   app.post('/authentication', handleResponse(async (req, res, next) => {
-    // body should contain {iv, cipherText, lookupKey}
-    let body = req.body
+    // body should contain {iv, cipherText, lookupKey, walletAddress}
+    const body = req.body
+    const libs = req.app.get('audiusLibs')
+
+    let score, ok, hostname
+    if (body.token) {
+      ({ score, ok, hostname } = await libs.captcha.verify(body.token))
+      req.logger.info(`CAPTCHA - Got captcha score: ${score}, is above minimum threshold score? ${ok}`)
+      if (!ok) return errorResponseBadRequest('CAPTCHA - Failed captcha')
+    } else {
+      req.logger.warn('CAPTCHA - No captcha found on request')
+    }
+
+    // Add recaptcha score and route context to table
+    try {
+      await models.RecaptchaScores.create({
```

Comment:
i dont think we should await on this since it's not critical for the success of this endpoint. my feeling would be to fire & forget calling the util